Assets/MixedRealityToolkit.Examples/Demos/UX/ManipulationHandler/Scripts/ChangeManipulation.cs

```diff
@@ -0,0 +1,46 @@
+﻿
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// Test script that forcefully stops manipulation on the manipulatedObject when it collides with the collisionTrigger
+    /// </summary>
+    public class ChangeManipulation : MonoBehaviour
+    {
+        public ManipulationHandler manipulatedObject;
+        public Collider collisionTrigger;
+
+        private Collider manipulatedObjCollider;
+
+        private void Start()
+        {
+            if (manipulatedObject != null)
+            {
+                manipulatedObjCollider = manipulatedObject.GetComponent<Collider>();
+            }
+        }
+
+        private void Update()
+        {
+            TryStopManipulation();
+        }
+
+        public void TryStopManipulation()
```

Comment:
Nit: could you use trigger volumes instead? like stop manipulation in [OnTriggerEnter](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnTriggerEnter.html)?

Reply:
looked into this, but I can't change it easily bc we're not receiving any of the collision events when we're manipulating the object - need to look into how to enable those events in another PR first